change/@ni-nimble-components-0c56aa86-23c7-4259-abc3-aaccccf9e8c6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I marked this as `patch` instead of `none` because I want to make sure beachball will publish a new `nimble-components` that consumes `nimble-tokens`. This may not be necessary (not sure how beachball decides between this field and the other package's `dependentChangeType`) but it didn't seem harmful.